phyui/plot/waveforms.py

```diff
@@ -325,6 +326,7 @@ class WaveformView(PanZoomCanvas):
     def __init__(self, **kwargs):
         super(WaveformView, self).__init__(**kwargs)
         self.visual = Waveforms()
+        self.resizable = Bool(defaultValue=True, sync=True)
```

Comment:
> IPython/utils/traitlets.py

you mean [this](https://github.com/ipython/ipython/blob/master/IPython/utils/traitlets.py#L321)?